dependencies.props

```diff
@@ -40,13 +45,22 @@
       <BuildInfoPath>$(BaseDotNetBuildInfo)standard/$(DependencyBranch)</BuildInfoPath>
       <CurrentRef>$(StandardCurrentRef)</CurrentRef>
     </RemoteDependencyBuildInfo>
+    <RemoteDependencyBuildInfo Include="WCF">
+      <BuildInfoPath>$(BaseDotNetBuildInfo)wcf/$(DependencyBranch)</BuildInfoPath>
+      <CurrentRef>$(WCFCurrentRef)</CurrentRef>
+    </RemoteDependencyBuildInfo>
 
     <DependencyBuildInfo Include="@(RemoteDependencyBuildInfo)">
       <RawVersionsBaseUrl>https://raw.githubusercontent.com/dotnet/versions</RawVersionsBaseUrl>
     </DependencyBuildInfo>
     
     <XmlUpdateStep Include="CoreFx">
       <Path>$(MSBuildThisFileFullPath)</Path>
+      <ElementName>CoreFxVersion</ElementName>
+      <PackageId>Microsoft.CSharp</PackageId>
```

Comment:
This should be `Microsoft.Private.CoreFx.NETCoreApp`.  The way this auto-update stuff works is it gets the new version from [https://github.com/dotnet/versions/blob/master/build-info/dotnet/corefx/master/Latest_Packages.txt#L4](https://github.com/dotnet/versions/blob/master/build-info/dotnet/corefx/master/Latest_Packages.txt#L4).  If the package hasn't been updated, then the version isn't updated.

So in servicing releases, where we only change `System.Net.Http`, we won't get a new `Microsoft.CSharp` package, but the private/transport package will be updated.  We want the version property to be updated whenever a new `Microsoft.Private.CoreFx.NETCoreApp` package is created.